source/geometries/Next100FieldCage.cc

```diff
@@ -72,8 +72,8 @@ Next100FieldCage::Next100FieldCage(G4double grid_thickn):
   // cathode_sapphire_wdw_dist_(257.8 * mm),
 
   // external to teflon (hdpe + rings + holders)
-  hdpe_tube_int_diam_ (1093. * mm - 11. * mm), // we keep the thickness, but we make it smaller to be closer to the staves
-  hdpe_tube_ext_diam_ (1105.4 * mm - 11. * mm),
+  hdpe_tube_int_diam_ (1081. * mm),
```

Comment:
Maybe it's irrelevant, but the thickness before  and after is not exactly the same.

Reply:
No, it's not the same. It turns out that the external hdpe cylinder is 6 mm-thick, while the internal one is 6.5 mm-thick, according to the drawing attached above.
Since my understanding is that the internal one has been kept, I think that now it is done correctly.

Reply:
Great!